compiler/p/codegen/PPCSystemLinkage.cpp

```diff
@@ -714,11 +714,12 @@ TR::PPCSystemLinkage::createPrologue(
       if (size > (-LOWER_IMMED))
          {
          cursor = loadConstant(cg(), firstNode, -size, gr11, cursor);
-         cursor = generateMemSrc1Instruction(cg(),TR::InstOpCode::Op_stux, firstNode, new (trHeapMemory()) TR::MemoryReference(sp, gr11, TR::Compiler->om.sizeofReferenceAddress(), cg()), sp, cursor);
+         cursor = generateMemSrc1Instruction(cg(),TR::InstOpCode::Op_stux, firstNode, TR::MemoryReference::createWithIndexReg(cg(), sp, gr11, TR::Compiler->om.sizeofReferenceAddress()), sp, cursor);
          }
       else
          {
-         cursor = generateMemSrc1Instruction(cg(),TR::InstOpCode::Op_stu, firstNode, new (trHeapMemory()) TR::MemoryReference(sp, -size, TR::Compiler->om.sizeofReferenceAddress(), cg()), sp, cursor);
+         //NOTE (remove this comment later): the fact that the displacement here is negative might mess things up when displacement is changed from 32 bits to 64 bits
```

Comment:
This comment is identical to one I pointed out in my previous review and should also be removed.